Modules/Scripted/DICOMLib/DICOMUtils.py

```diff
@@ -78,16 +78,16 @@ def openDatabase(databaseDir):
 
 #------------------------------------------------------------------------------
 # Open temporary DICOM database, return current database directory
-def openTemporaryDatabase(directory=None, absolutePath=False):
+def openTemporaryDatabase(directory=None, isAbsolutePath=False):
   if not hasattr(slicer,'dicomDatabase') or not hasattr(slicer.modules,'dicom'):
```

Comment:
I wonder if we could remove this extra parameter and use https://docs.python.org/2/library/os.path.html#os.path.isabs